certbot/main.py

```diff
@@ -1112,6 +1112,8 @@ def renew_cert(config, plugins, lineage):
         installer.restart()
         notify("new certificate deployed with reload of {0} server; fullchain is {1}".format(
                config.installer, lineage.fullchain), pause=False)
+        # Run deployer
+        updater.run_renewal_deployer(renewed_lineage, installer, config)
```

Comment:
Putting this both here and in `handle_renewal_request` calls it twice during the same run. `main.renew` calls `renewal.handle_renewal_request` calls `main.renew_cert`

Reply:
I believe the functionality is correct, as this calls the `interfaces.RenewDeployer.renew_deploy()` and `renewal.handle_renewal_request`calls `interfaces.GenericUpdater.generic_updates()` through `updater.run_renewal_updaters()`. And I don't think either of them gets called twice.